src/util/tubemap.js

```diff
@@ -3702,10 +3703,22 @@ function trackDoubleClick() {
 
 function trackSingleClick() {
   /* jshint validthis: true */
-  const trackName = d3.select(this).attr("trackName");
+  const trackID = d3.select(this).attr("trackID");
+  let current_track = tracks[trackID];
```

Comment:
Something is wrong here somehow; when I click on a read I get a popup for a *different* read, with the name and CIGAR and other info for some other read. I think maybe we are doing something like sorting the reads between when the "trackID" D3 attribute is set and when we read it out and get the track? Or maybe what we put in isn't the right thing?

Reply:
@shreyasun I think I fixed this by just brute force scanning for the right track.